SFBaseKitDemo/SFBaseKitDemo/ExampleLoginModule/LoginCoordinator.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LogingCoordinator.swift
+//  SFBaseKitDemo
+//
+//  Created by Vesela Ilchevska on 15.04.20.
+//  Copyright © 2020 Upnetix. All rights reserved.
+//
+
+import Foundation
+import SFBaseKit
+
+protocol LoginSceneDelegate: class {
+    func sceneShouldContinueToForgottenPassword()
+    func sceneShouldContinueToLoginIn()
+}
+
+class LoginCoordinator: Coordinator {
+    
+    // MARK: - Properties
+    private var navigationController: UINavigationController?
```

Comment:
Having this in the `init` method, means that this property could be `let` and non-optional. Furthermore on this topic, you might want to make it `unowned` too for memory management purposes.

Reply:
Cool! Done